DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -802,10 +829,28 @@ public async Task<IReadOnlyList<DiscordMember>> SearchMembersAsync(string name,
     /// <exception cref="NotFoundException">Thrown when the <paramref name="user"/> or <paramref name="accessToken"/> is not found.</exception>
     /// <exception cref="BadRequestException">Thrown when an invalid parameter was provided.</exception>
     /// <exception cref="ServerErrorException">Thrown when Discord is unable to process the request.</exception>
-    public async Task<DiscordMember?> AddMemberAsync(DiscordUser user, string accessToken, string nickname = null, IEnumerable<DiscordRole> roles = null,
+    public async Task<DiscordMember?> AddMemberAsync(DiscordUser user, string accessToken, string? nickname = null, IEnumerable<DiscordRole>? roles = null,
         bool muted = false, bool deaf = false)
         => await Discord.ApiClient.AddGuildMemberAsync(Id, user.Id, accessToken, muted, deaf, nickname, roles);
 
+    /// <summary>
+    /// Adds a new member to this guild
+    /// </summary>
+    /// <param name="userId">The id of the User to add</param>
+    /// <param name="accessToken">User's access token (OAuth2)</param>
+    /// <param name="nickname">new nickname</param>
+    /// <param name="roles">new roles</param>
+    /// <param name="muted">whether this user has to be muted</param>
+    /// <param name="deaf">whether this user has to be deafened</param>
+    /// <returns>Only returns the member if they were not already in the guild</returns>
+    /// <exception cref="UnauthorizedException">Thrown when the client does not have the <see cref="DiscordPermissions.CreateInstantInvite" /> permission.</exception>
+    /// <exception cref="NotFoundException">Thrown when the <paramref name="userId"/> or <paramref name="accessToken"/> is not found.</exception>
+    /// <exception cref="BadRequestException">Thrown when an invalid parameter was provided.</exception>
+    /// <exception cref="ServerErrorException">Thrown when Discord is unable to process the request.</exception>

```

Comment:
doc comments should be full sentences, ending in a period

Reply:
Currently not in the mood to rewrite all xmldocs in that file :when: